packages/functional-tests/tests/settings/avatar.spec.ts

```diff
@@ -68,3 +78,19 @@ test.describe('severity-1 #smoke', () => {
     );
   });
 });
+
+async function signInAccount(
```

Comment:
Noticed this block of code was used in other files as well, I think the duplication is fine.